src/components/ComponentPages/NickName/NickNameUI/index.tsx

```diff
@@ -25,35 +25,34 @@ export default function NickNameUI({
   editNickName,
   nickNameForm,
   handleAddNickName,
-  handleNickNameCancel
+  handleNickNameCancel,
 }) {
-
   const [data, setData] = useState(originData);
   const columns = [
     {
-      title: 'Sr',
-      dataIndex: 'srNo',
-      width: '5%',
+      title: "Sr",
```

Comment:
Not sure why `'` replaced with `"`. JS treats `'` faster than `"`. Same for all

Reply:
it's changed by formatter.